rocketpy/Environment.py

```diff
@@ -367,6 +367,7 @@ def __init__(
             self.setDate(date, timeZone)
         else:
             self.date = None
+            self.datetime_date = None
```

Comment:
The `datetime_date` variable could have be introduced directly to the `beta/v1.0.0`, avoiding future dilemmas

Reply:
True, Ill make a PR for this

Reply:
> True, Ill make a PR for this

Please don't forget to explain your motivation (problem + solution) in the PR description. Again, avoiding future stress.